### PR TITLE
Refactor: config thingy

### PR DESCRIPTION
**Pull request by TonybynMp4**
## Description

i hope it doesn't break everything
## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

**Comment by mafewtm** [comment]
This 100% breaks. It runs into circular dependencies because someone thought it was a good idea to modularize the ENTIRE script. That's why I've been slowly breaking it down and merging them all into their own categories. It just hasn't been finished it yet

**Comment by TonybynMp4** [comment]
> This 100% breaks. It runs into circular dependencies because someone thought it was a good idea to modularize the ENTIRE script. That's why I've been slowly breaking it down and merging them all into their own categories. It just hasn't been finished it yet

Blame jorn 😆, pretty sure it worked for me tho so that's weird

**Comment by mafewtm** [comment]
> > This 100% breaks. It runs into circular dependencies because someone thought it was a good idea to modularize the ENTIRE script. That's why I've been slowly breaking it down and merging them all into their own categories. It just hasn't been finished it yet
> 
> Blame jorn 😆, pretty sure it worked for me tho so that's weird

I mean I'm pretty sure it runs into circular dependency issues. try it all out, open the menus, and just double check I guess

**Comment by TonybynMp4** [comment]
> I mean I'm pretty sure it runs into circular dependency issues. try it all out, open the menus, and just double check I guess

Everything works fine for me 👍🏻

File: client/menus/colors.lua [hunk 1]
```diff
@@ -6,21 +6,23 @@ local originalWindowTint
 local originalLabelIndex = 1
 local originalInterior
 local originalLivery = {}
+local config = require('config.client')
+local sharedConfig = require('config.shared')
```
Comment (BerkieBb) [inline client/menus/colors.lua]: You don't need the () around it, function calls work without it and we use it without in these cases

File: fxmanifest.lua [hunk 2]
```diff
@@ -27,6 +26,7 @@ ui_page 'web/index.html'
 
 files {
     'client/**/*.lua',
+    'config/*.lua',
```
Comment (BerkieBb) [inline fxmanifest.lua:28]: Load the configs before client files

File: client/menus/colors.lua (hunk 1 above)
Comment (TonybynMp4) [reply client/menus/colors.lua]: i blame copilot 😆

**Review by BerkieBb — APPROVED** [review]

**Review by mafewtm — APPROVED** [review]

**Pull request merged**